lake/seekindex/entry.go

```diff
@@ -0,0 +1,53 @@
+package seekindex
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/vector"
+)
+
+type Entry struct {
+	Min    zed.Value `zed:"min"`
+	Max    zed.Value `zed:"max"`
+	ValOff uint64    `zed:"val_off"`
+	ValCnt uint64    `zed:"val_cnt"`
+	Offset uint64    `zed:"offset"`
+	Length uint64    `zed:"length"`
+}
+
+func (e Entry) Range() Range {
+	return Range{
+		Offset: int64(e.Offset),
+		Length: int64(e.Length),
+		ValOff: int64(e.ValOff),
+		ValCnt: int64(e.ValCnt),
+	}
+}
+
+type Entries []Entry
+
+func (entries Entries) Filter(b *vector.Bool) Entries {
```

Comment:
call this Lookup?  if you changed the name of Entries, then the naming would make more sense, e.g., do a lookup on an Index, e.g., `func (i Index) Lookup(b *vector.Bool) []Range`